From 20a9024d3dda9786e21a14b796119f1294886e12 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Tue, 27 Jun 2017 11:36:43 +0800
Subject: [PATCH] drm: bridge: dw-hdmi: correct the judgment condition of
 pll_config

The judgment condition of pll_config is phy_config->mpixelclock. It is
a hand mistake of commit <594a077e31e8b1ff0bd192e9af702fecaeea31ba>.

Change-Id: I7e312fd3b7982ca4fdf610e577776bc5a45c4bde
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/gpu/drm/bridge/dw-hdmi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/dw-hdmi.c b/drivers/gpu/drm/bridge/dw-hdmi.c
index 1ad15af799a3..47b672256003 100644
--- a/drivers/gpu/drm/bridge/dw-hdmi.c
+++ b/drivers/gpu/drm/bridge/dw-hdmi.c
@@ -1419,7 +1419,7 @@ static int hdmi_phy_configure_dwc_hdmi_3d_tx(struct dw_hdmi *hdmi,
 			break;
 
 	for (; phy_config->mpixelclock != ~0UL; phy_config++)
-		if (mpixelclock <= mpll_config->mpixelclock)
+		if (mpixelclock <= phy_config->mpixelclock)
 			break;
 
 	if (mpll_config->mpixelclock == ~0UL ||
-- 
2.35.3

